core/server/data/schema/fixtures/fixtures.json

```diff
@@ -485,6 +539,24 @@
             "entries": {
                 "Administrator": {
                     "db": "all",
+                    "mail": "all",
+                    "notification": "all",
+                    "post": "all",
+                    "setting": "all",
+                    "slug": "all",
+                    "tag": "all",
+                    "theme": "all",
+                    "user": "all",
+                    "role": "all",
+                    "client": "all",
+                    "subscriber": "all",
+                    "invite": "all",
+                    "redirect": "all",
+                    "webhook": "all",
+                    "integration": "all",
+                    "api_key": "all"
+                },
+                "Admin Integration": {
```

Comment:
One last question to the permissions. Not sure if i have asked this already.
This role has a lot of permissions. Is that correct? 🤔

Reply:
Yes, it should be the same access as Administrator except locked down a little more so that it can't be used to give itself more roles/permissions, alter permissions of any other user, or download/drop the database.

Reply:
> download/drop the database.

👍 Yeah i saw that this was already removed, which is nice 👍 

--

What about:

delete a post? 🤔 
delete a user? isn't it weird that an integration can delete editors+authors?

I am just double checking with you :)

Reply:
We were thinking how you'd want to use the API if you wanted to integrate an external auth system, or if you are pulling content in from somewhere and need to create new authors on the fly.

The original discussion was that an integration would be tied to a user with Administrator privileges until we realised that would likely give them a little too much power and also cause issues if users are demoted or deleted. With the role we wanted the same level of access but just restricted a little so that credentials can't be leaked. There's another PR coming later that deals with the per-model `permissible` functions that lock things down a bit further.

Reply:
Yeah all of this makes sense, i am just wondering about some particular restrictions.

> There's another PR coming later that deals with the per-model permissible functions that lock things down a bit further.

Is there a branch already?

Reply:
There is code in https://github.com/TryGhost/Ghost/pull/9869 waiting to be extracted, there were quite a lot of changes in this PR and the authentication PR though so I was holding off until the they're closer to being merged.

Reply:
Okay it makes sense to extend some of the permissible model functions for specific checks. 

But we want to keep the permission for e.g. delete post or delete user for the integration role?

Reply:
Yes, so far the spec has been been integrations having Administrator-level access.

Reply:
Thanks. If that was confirmed, i am fine with leaving as is :)